src/pages/Work.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import Navbar from '../components/navbarWork';
+import SideBar from '../components/Sidebar';
+import MxGraph from '../components/MxGraph';
+import { Grid } from 'semantic-ui-react';
+
+class Workspace extends React.Component<any, any> {
+
+    constructor(props: any) {
+        super(props);
+        this.state = {menuVisible: false};
+    }
+
+    render() {
+        return (
+            <Grid style={{height: '100vh'}}>
+                <Grid.Row style={{height: '10%', paddingBottom: 0}}>
+                    <Grid.Column>
+                        <Navbar/>
+                    </Grid.Column>
+                </Grid.Row>
+
+                <Grid.Row columns={16} style={{height: '90%', paddingTop: 0}} stretched={true}>
+                    <Grid.Column width={2} style={{paddingRight: 0}}>
+                        <SideBar/>
+                    </Grid.Column>
+                    <Grid.Column width={14} style={{paddingLeft: 0}}>
+                        <MxGraph/>
+                    </Grid.Column>
+                </Grid.Row>
+            </Grid>
+        )
+            ;
```

Comment:
Interestingly placed semicolon. :)

I've seen this pattern pop up lots of times in this PR. Is this a thing people do? All the [examples](https://github.com/mlaursen/react-md/blob/b0ba42a8dd5fda6f3e6bca21f5de90f67a73d53d/examples/with-typescript/src/App.tsx#L24) I've seen so far use the admittedly rather sad-looking `);`, but I'm not that experienced with React.

Reply:
Check.

Reply:
👍